modloader.py

```diff
@@ -9,7 +9,7 @@ def load_modules(updater):
     api = TelegramAPI(updater)
 
     for module_name in listdir('./modules'):
-        if module_name.endswith('.py'):
+        if not module_name.startswith('lib-') and module_name.endswith('.py'):
```

Comment:
to not be shitty and have libs if they are needed. Or just use it to disable module 😄.